framework/src/st_things/things_stack/framework/things_data_manager.c

```diff
@@ -2275,17 +2297,17 @@ int things_get_resource_type(const char *resource_uri, int *count, char ***resou
 	return 0;
 }
 
-const int dm_get_wifi_property_interface(void)
+const int dm_get_wifi_property_interface()
```

Comment:
Could you explain why did you remove void ?

Reply:
Its handle in another PR #2544 . Because of this I remove it.